src/pykiso/lib/connectors/cc_pcan_can.py

```diff
@@ -143,22 +152,15 @@ def __init__(
         generic logfile in the current working directory, which will be
         overwritten every time, a log is initiated.
         """
+        if self.trace_path.is_file():
+            self.trace_path = self.trace_path.parent
+            log.warning(
+                f"Input path is a file. Trace will be save into following path {self.trace_path}"
```

Comment:
why warning ? Sounds that we don't suggest to use logfile name

Reply:
The reason is that the PCAN Basic API doesn't allow to specify a file name for the trace, hence the warning
Maybe be a bit more specific in your warning message e.g. `f"File names are not supported for trace file creation. Trace will be written to {self.trace_path}"`